portal/src/RoleAndGroupsLayout.tsx

```diff
@@ -9,8 +10,13 @@ import {
   ErrorMessageBarContextProvider,
 } from "./ErrorMessageBar";
 
-interface RoleAndGroupsLayoutProps {
+export interface RoleAndGroupsHeaderProps {
   breadcrumbs: BreadcrumbItem[];
+  subitem?: JSX.Element | null;
+}
+
+interface RoleAndGroupsLayoutProps {
+  header: RoleAndGroupsHeaderProps;
```

Comment:
Usually we won't nest props, because this make it difficult to use (Because inline objects doesn't not have a stable reference, usually you need to use `useMemo` to create the object).

Prefer flatten the props and name it `headerBreadcrumbs` and `headerSubItems`